lib/matplotlib/tests/test_artist.py

```diff
@@ -187,6 +189,12 @@ def test_properties():
         assert len(w) == 0
 
 
+@cleanup
```

Comment:
This test alone should be reason enough to stick with your PR. You can add the example I gave in my PR if you want something less contrived.